build.sbt

```diff
@@ -44,51 +44,52 @@ inThisBuild(
 )
 
 lazy val V = new {
-  val munit = "1.0.0-M8" // "0.7.29"
-  val munitZio = "0.1.1"
+  val munit = "1.0.0-RC1" // "0.7.29"
+  val munitZio = "0.2.0"
 
   // https://mvnrepository.com/artifact/dev.zio/zio
-  val zio = "2.0.18"
+  val zio = "2.0.22"
   val zioConfig = "4.0.1"
-  val zioLogging = "2.1.16"
-  val zioJson = "0.3.0"
-  val zioHttp = "3.0.0-RC4"
-  val zioCatsInterop = "3.3.0"
-  val zioMetricsConnector = "2.1.0"
-  val zioMock = "1.0.0-RC11"
-  val mockito = "3.2.16.0"
-  val monocle = "3.1.0"
+  val zioLogging = "2.1.17"
+  val zioJson = "0.6.2"
+  val zioHttp = "3.0.0-RC6"
+  val zioCatsInterop = "3.3.0" // TODO "23.1.0.2" // https://mvnrepository.com/artifact/dev.zio/zio-interop-cats
+  val zioMetricsConnector = "2.3.1"
+  val zioMock = "1.0.0-RC12"
+  val mockito = "3.2.18.0"
+  val monocle = "3.2.0"
 
   // https://mvnrepository.com/artifact/io.circe/circe-core
-  val circe = "0.14.6"
+  val circe = "0.14.7"
 
-  val tapir = "1.6.4"
+  val tapir = "1.6.4" // scala-steward:off // TODO "1.10.5"
+  val http4sBlaze = "0.23.15" // scala-steward:off  // TODO "0.23.16"
 
-  val typesafeConfig = "1.4.2"
+  val typesafeConfig = "1.4.3"
   val protobuf = "3.1.9"
-  val testContainersScala = "0.41.0"
-  val testContainersJavaKeycloak = "3.2.0" // scala-steward:off
+  val testContainersScala = "0.41.3"
+  val testContainersJavaKeycloak = "3.3.0" // scala-steward:off
```

Comment:
Better to stick to 3.2. Version 3.3 uses keycloak v24 which is incompatible and require code changes in container bootstrapping.

https://github.com/dasniko/testcontainers-keycloak/releases/tag/v3.3.0